x-pack/plugins/cases/public/components/custom_fields/configure_toggle_field.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { UseField } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
+
+import type { CustomFieldBuilder } from './types';
+import { CustomFieldTypes } from './types';
+import { FieldOptionsSelector } from './field_options/field_options_selector';
+
+export const configureToggleCustomFieldBuilder: CustomFieldBuilder = () => ({
+  build: () => [
+    {
+      // eslint-disable-next-line react/display-name
+      ConfigurePage: () => (
+        <UseField
+          path="fieldOptions"
```

Comment:
Having two fields with the same path may cause one to override the other. We can either use the `UseArray` component or rename the path to `text.fieldOptions`.

Reply:
We discussed it offline and it is ok as it is. I was confused.